packaging/installer/netdata-updater.sh

```diff
@@ -460,8 +460,10 @@ if [ "${IS_NETDATA_STATIC_BINARY}" == "yes" ]; then
 
   echo "${install_type}" > /opt/netdata/etc/netdata/.install-type
 
-  echo >&2 "Switching back to ${PREVDIR}"
-  cd "${PREVDIR}"
+  if [ -e "${PREVDIR}" ]; then
+    echo >&2 "Switching back to ${PREVDIR}"
+    cd "${PREVDIR}"
+  fi
 else
```

Comment:
I don't really know why we need to `cd ${PREVDIR}` at all btw 🤷‍♂️ Perhaps we can just remove it. But it least it is harmless with `-e` test.

Reply:
push/popd  seem more appropriate here.

Reply:
Thanks for the suggestion, but I have never used these shell built-ins. Can you elaborate on why they seem more appropriate to you?

Also, I'm fixing a very specific problem in this PR. I think switching to pushd is out of its context.

Reply:
Definitely. It was just a remark for the future.

Why `pushd`/`popd` seem more appropriate? Well, you may consider them as a 'native' way to perform navigation in the shell scripts. It is more concise.

Reply:
`pushd`/`popd` also aren’t part of POSIX, so we can’t really rely on them.

As far as why this particular bit is here, it’s to cover a particular edge case of a user sourcing the script. Without it in that case, they end up left in the working directory of the updater script instead of where they started.

Reply:
Hm... I know them for so long time that I was sure they are the part of initial sh 😞

Reply:
They’re a very common set of extensions, but they aren’t part of POSIX.

Reply:
@iigorkarpov consider approving this PR 😉

Reply:
Didn't I? It was just a remark, I never meant to slow it down.